src/entities/inputValidator/inputValidator.ts

```diff
@@ -107,14 +107,18 @@ export class InputValidator {
 
     private validateChain(chainId: number): void {
         if (chainId in ChainId) return;
-        throw new Error(`Unsupported ChainId: ${chainId}`);
+        throw new SDKError(
+            'Input Validation',
+            'Any',
+            `Unsupported chainId: ${chainId}`,
+        );
```

Comment:
What is the initial reason to throw via `SDKERROR` instead of an imported one from utils?

Reply:
same as above